server/webserver.go

```diff
@@ -132,10 +132,10 @@ func (s *LivepeerServer) cliWebServerHandlers(bindAddr string) *http.ServeMux {
 			pNames = append(pNames, p.Name)
 		}
 		config := struct {
-			BroadcastConfig    *BroadcastConfig
+			MaxPrice           *big.Rat
 			TranscodingOptions string
 		}{
-			BroadcastCfg,
+			BroadcastCfg.MaxPrice(),
```

Comment:
This could just be `price`?

Reply:
Not the same scope. Could define price at the top of that function rather than in the mux block for `setBroadcastConfig `but seems less clean to do it that way imo.

Reply:
Oh I see, my mistake - I was reading the collapsed diff on GH and didn't realize that this was in its own handler 😓